<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>359</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>359</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SV Paciente Uegencia segun Fecha Consulta</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_texto">
        <property name="text">
         <string>Fecha Consulta:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Carnet Paciente:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="carnet_paciente"/>
      </item>
      <item row="0" column="1">
       <widget class="QDateEdit" name="fecha_consulta">
        <property name="displayFormat">
         <string>d/M/yyyy</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btn_accion">
        <property name="text">
         <string>Buscar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_cancelar">
        <property name="text">
         <string>Cancelar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
